docs/_temp/HowToOrganizeDevlakeProjects.md

```diff
@@ -0,0 +1,189 @@
+
+#### 4.1.3. Configuring Connections
+This part is described in [GitHub](/docs/Configuration/GitHub.md) connection configuration.
+For other platforms check the [Configuration Guide](/docs/Configuration)
+
+### 4.1.4 Using connections
+
+At this point, we have projects and connections created on platform DevLake. 
+It is time to bind those connections to the projects. To do so, follow the steps
+described in the [Tutorial](/docs/Configuration/Tutorial.md).
+
+### 4.1.5 Resulting Metrics
+
+To know if the data of a project is successfully collected go to your DORA Dashboard:
+
+![](navigate_to_dora_1.png)
+![](navigate_to_dora_2.png)
+
+If everything goes well, you should see all the 4 charts.
+If something is wrong, and you are puzzled why, check out the
+[Debugging Dora Issue Metrics](docs/Troubleshooting/Dashboard.md#debugging-dora-issue-metrics) page.
+
+#### 4.1.6. How can I observe metrics by project?
+In the same DORA dashboard check out this menu point:
+![](observe_metrics_by_project_panel.png)
+
+The metrics should change when you select or deselect projects, representing the projects you selected.
+
+### 4.2. Use Case 2
+
+Each community or company have their own flavor on how they work. In this example let's assume a company that has
+multiple teams, each working on one or more projects.
+
+To keep it succinct, let's have a look at 2 teams that are called _Payments_ and _Internal Tools_.
+Payments team works with only one `project` also called "payments", while Internal Tools deals with 2 `projects`
+called "it-legacy" and "it-new" (such naming is to keep it simple).
+Each `project` consists of various `repos`, and `cicd pipelines`.
+Also, each team uses their own `board` making the total of 2.
+Some Internal Tools `repos` are shared between projects.
+
+Note: Assume that both teams work with GitHub and Jenkins. Also assume that payments team works with Jira boards,
+while internal tools team needs to use a **webhook** to publish their incidents.
+
+Following diagram describes the situation.
+
+![](project_use_case_2.png)
+
+#### 4.2.1. Organizing Projects
+DORA is good for seeing the effects of the new changes in team's methodology.
+However, for DORA itself **there are no teams**, only projects. 
+The `team` concept is redundant and only introduces noise and complexity, for no benefit.
+
+That said, we have 3 `projects` to create in DevLake: payments, it-legacy, and it-new
+
+Note: DevLake charts let you combine data between projects, so to have most flexibility it is always better to keep
+the `projects` **atomic**.
+
+#### 4.2.2. Adding Connections
+
+For payments `project` this is straightforward, just create: 
+- 1 connection for all `repos` to gather `pull requests`,
+- 1 connection for all `deployments`
+- 1 connection to Jira to gather `incidents`
+
+For it-legacy `project`:
+- 1 connection for `repos` _it-legacy-1_ and _it-legacy-1_
```

Comment:
Normally, we don't recommend users to create multiple connections for a shared data scope, for instance, GitHub repos. This will increase the time to collect the data because the shared repos will be stored in multiple copies in the DB. See pic:
![image](https://github.com/apache/incubator-devlake-website/assets/14050754/1e3c2b56-9b31-4a65-bbdb-92d44151a9fd)



Instead, we encourage users to create ONE connection and add all data scopes (p-1, p2, ... , p-10, it-legacy-1, it-legacy-2) in the Data Connection page, and when users:
- bind this connection to `Project payments`, they only need to choose 'p-1, p2, ... , p-10'
- bind this connection to `Project it-legacy`, they only need to choose 'it-legacy-1, it-legacy-2'
In this way, the data of 'p-1, p2, ... , p-10, it-legacy-1, it-legacy-2' will only be stored in a single copy. That is to say, the sync-up time will not be doubled/tripled either.

![image](https://github.com/apache/incubator-devlake-website/assets/14050754/8e41bdcb-ce91-46e7-980f-33a2d915e120)

So, in my opinion, there should be a '4.2.0. Create Data Connections' before 4.2.1. And the current '4.2.2. Adding Connections' should be renamed to '4.2.2. Adding Connections and repos to Projects'

Reply:
That's a great point. In fact, I think it's good to mention that in the guide.